src/screens/home/Home.js

```diff
@@ -302,6 +302,13 @@ class Home extends Component {
 												width="100%"
 												height="50%"
 											/>
+											{/*TODO: Investigate why the SVG does not show up unless
+											    other elements are present to force more space on
+											    the screen. As a temportary fix, we added in a
+											    a bunch of text components. Depending on the device
+											    and size of the scree, some of the text components
+											    will appear below the SVG.
+											*/}
```

Comment:
I'd definitely create a GitHub issue for this too. For right now, I'd just paste this TODO into the issue description and add a link to this code block. We can discuss format for GH issues at our next meeting.